java/src/plume/BCELUtil.java

```diff
@@ -494,8 +496,14 @@ public static void add_line_numbers(MethodGen mg, InstructionList il) {
   public static void setup_init_locals(MethodGen mg) {
 
     // Get the parameter types and names.
-    Type[] arg_types = mg.getArgumentTypes();
-    String[] arg_names = mg.getArgumentNames();
+    @SuppressWarnings(
+        "nullness") // The initialization checker warns here when the SameLen annotations are added.
```

Comment:
Can you make the comment explain why the code is correct?  (Same comment a few other places.)